.github/actions/image-retag/action.yml

```
@@ -0,0 +1,22 @@
+runs:
+  using: composite
+  steps:
+    - name: Pull docker images
+      shell: bash
+      run: docker pull ${{ inputs.image_source }}
+    - name: Tag docker images
+      shell: bash
+      run: docker tag ${{ inputs.image_source }} ${{ inputs.image_target }}
+    - name: Push docker images
+      shell: bash
+      run: docker push ${{ inputs.image_target }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)